OpenEdXMobile/src/main/java/org/edx/mobile/http/notifications/EdxCourseAccessErrorState.kt

```diff
@@ -0,0 +1,79 @@
+package org.edx.mobile.http.notifications
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.LayoutInflater
+import android.widget.FrameLayout
+import androidx.core.content.ContextCompat
+import org.edx.mobile.R
+import org.edx.mobile.databinding.LayoutCourseAccessErrorBinding
+import org.edx.mobile.extenstion.setVisibility
+
+class EdxCourseAccessErrorState @JvmOverloads constructor(
+    context: Context, attrs: AttributeSet? = null, defStyleAttr: Int = 0
+) : FrameLayout(context, attrs, defStyleAttr) {
+
+    private val layout: LayoutCourseAccessErrorBinding =
+        LayoutCourseAccessErrorBinding.inflate(LayoutInflater.from(context))
+
+    init {
+        this.addView(layout.root)
+        attrs?.let {
+            val typedArray = context.obtainStyledAttributes(it, R.styleable.StateLayout)
```

Comment:
IMO, better to define separate attribute for this custom View

Reply:
done.